### DISCUSSION TITLE
How to update/delete the contact image associated with a given contact?

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#17464] [How to] [answered]
How can I change/update the contact image associated with a contact when there is already a contact image associated? When I select a different contact image file and try to save the contact, I get the following error (tested on demo.yetiforce.com):
![q](https://github.com/YetiForceCompany/YetiForceCRM/assets/10230144/24be74e2-a77a-4b61-bf83-3134598391a3)

Similarly, how can I delete a contact image associated with a contact?

**Comment by rskrzypczak** [comment, marked as answer]
It's quite simple, just click on the image and then delete it ;) 
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/10221999/aaabac18-883a-4373-a2b2-44238da3a6aa)
